modules/nodes-nodes-audit-log-basic-viewing.adoc

```diff
@@ -69,7 +69,7 @@ ip-10-0-170-165.ec2.internal audit.log
 .. View the Kubernetes API server log for a specific master node and timestamp or view all the logs for that master:
 +
 ----
-$ oc adm node-logs <node-ip> <log-name> --path=kube-apiserver/<log-name>
+$ oc adm node-logs <node-ip> --path=kube-apiserver/<log-name>
```

Comment:
Same question here about whether `audit-2019-04-09T14-07-27.129.log` and `audit.log` should be removed from the examples below?

Reply:
Fixed.